mavedb_mapping/main.py

```diff
@@ -0,0 +1,13 @@
+from mavedb_mapping.metadata_process import metadata_obtain
+from mavedb_mapping.blat_alignment import mave_to_blat
+from mavedb_mapping.transcript_selection import main
+from mavedb_mapping.vrs_mapping import vrs_mapping
+from mavedb_mapping import data_file_path
+
+
+def main_map(urn):
```

Comment:
could you give a bit more description (maybe in a docstring?) about what the `urn` variable is here?